modules/lotamePanoramaIdSystem.js

```diff
@@ -29,6 +29,7 @@ const DAY_MS = 60 * 60 * 24 * 1000;
 const MISSING_CORE_CONSENT = 111;
 const GVLID = 95;
 const ID_HOST = 'id.crwdcntrl.net';
+const SAFARI_ID_HOST = 'c.ltmsphrcl.net';
```

Comment:
is this a cat and mouse game, are you rotating the domain everytime you get blacklisted?

Reply:
This change is to isolate requests that may try to set 3rd party cookies under certain conditions from those that will not even attempt to do so, and we do not anticipate further changes to either domain. To clarify that intent, we've changed the variable name to ID_HOST_COOKIELESS